dockers/docker-orchagent/start.sh

```diff
@@ -12,6 +12,18 @@ function start_app {
     done
 }
 
+function config_acl {
+    if [ -f "/etc/sonic/acl.json" ]; then
+        mkdir -p /etc/swss/config.d/acl
+        rm -rf /etc/swss/config.d/acl/*
+        translate_acl -m /etc/sonic/minigraph.xml -o /etc/swss/config.d/acl /etc/sonic/acl.json
+        for filename in /etc/swss/config.d/acl/*.json; do
+            [ -e "$filename" ] || break
```

Comment:
If there is no file in `/etc/swss/config.d/acl/`, items in `for` loop with be executed once with `filename` equals to `'/etc/swss/config.d/acl/*'`. And it will lead to error in swssconfig.